src/app/projects/projects-list/projects-list.component.html

```diff
@@ -200,6 +200,45 @@
       </tr>
     </tbody>
   </table>
+
+  <ng-container *ngIf="currentPage$ | async as currentPage">
```

Comment:
I think it would be cleaner to have this as a component.
Also, it might be possible to use the [angular-material one](https://material.angular.io/components/paginator).